### DISCUSSION TITLE
Signal integration - some characters are wrong...

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#766] [General]
I just did Signal integration. I get messages but sometimes they are not well formated - as per screenshot.
![image](https://user-images.githubusercontent.com/58336955/208913629-683becc7-4bec-4cd9-9e9e-923ee0a018dd.png)

I use "==>" in both names. First on the top looks good but the bottom one is somehow scrambled...
Is it something you can look into?
Paul

**Comment by cuu508** [comment]
Thanks for reporting, that's a bug – filed #767
